src/compounds/BlankRequestForm/BlankRequestForm.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Button,
+  Form,
+  Row,
+  Col,
+} from 'react-bootstrap'
+import Title from '../../components/Title/Title'
+import ShippingDetails from './ShippingDetails'
+import AdditionalInfo from './AdditionalInfo'
+
+const BlankRequestForm = () => {
```

Comment:
```suggestion
const BlankRequestForm = ({ onSubmit }) => {
```

Reply:
I see you deleted this prop [here](https://github.com/scientist-softserv/webstore-component-library/pull/81/commits/069351cfc7f379985224cba35f18bfa6a244b654), but without it...we can't return our data to the webstore.

Reply:
ok for sure. I wasn't sure if we still needed it but that makes sense